### ISSUE TITLE
Finetuning my own pre-trained RoBERTa on a custom classification task

### ISSUE THREAD

**Issue by NicolasMontes** [pytorch/fairseq#1728]
Hi, I pre-trained my own languaje model with the help on “Pretraining RoBERTa using your own data”. 
Then i want to use it to a clasification task with the help of “Finetuning RoBERTa on a custom classification task”.

In the example there are an argument “--arch roberta_large” that use a pre-trained model,  but I want to use my own model. How can I set the parameters to use
my own pre-trained model? 

TOTAL_NUM_UPDATES=10  
WARMUP_UPDATES=469     
LR=1e-05                
HEAD_NAME=imdb_head   
NUM_CLASSES=2           
MAX_SENTENCES=1       

CUDA_VISIBLE_DEVICES=0 python train.py ../DRUPAL-bin/ \
    --max-positions 512 \
    --max-sentences $MAX_SENTENCES \
    --max-tokens 4400 \
    --task sentence_prediction \
    --reset-optimizer --reset-dataloader --reset-meters \
    --required-batch-size-multiple 1 \
    --init-token 0 --separator-token 2 \
    --arch roberta \
    --criterion sentence_prediction \
    --classification-head-name $HEAD_NAME \
    --num-classes $NUM_CLASSES \
    --dropout 0.1 --attention-dropout 0.1 \
    --weight-decay 0.1 --optimizer adam --adam-betas "(0.9, 0.98)" --adam-eps 1e-06 \
    --clip-norm 0.0 \
    --lr-scheduler polynomial_decay --lr $LR --total-num-update $TOTAL_NUM_UPDATES --warmup-updates $WARMUP_UPDATES \
    --fp16 --fp16-init-scale 4 --threshold-loss-scale 1 --fp16-scale-window 128 \
    --max-epoch 10 \
    --best-checkpoint-metric accuracy --maximize-best-checkpoint-metric \
    --truncate-sequence \
    --find-unused-parameters

**Comment by myleott** [comment]
You're missing the `--restore-file` argument, please refer to the command here: https://github.com/pytorch/fairseq/blob/master/examples/roberta/README.custom_classification.md#5-run-training

In particular you need to add back `--restore-file $ROBERTA_PATH` to your training command (where ROBERTA_PATH will point to your pretrained checkpoint).

**Comment by NicolasMontes** [comment]
Thank you ... If I want to run it in cpu ... what other parameter do I have to specify in addition to `--cpu`?

**Comment by myleott** [comment]
I believe `--cpu` is all you need
